src/deroff.rs

```diff
@@ -1011,15 +952,35 @@ impl Deroffer {
         }
     }
 
-    fn not_whitespace(s: &str, idx: usize) -> bool {
+    /// AKA `prch`
+    fn not_whitespace(&self, idx: usize) -> bool {
         // # Note that this return False for the empty string (idx >= len(self.s))
         // ch = self.s[idx:idx+1]
         // return ch not in ' \t\n'
         // TODO Investigate checking for ASCII whitespace after mvp
-        s.get(idx..=idx)
-            .map(|string| " \t\n".contains(string))
+        self.s
+            .get(idx..=idx)
+            .map(|string| !" \t\n".contains(string))
             .unwrap_or_default()
     }
```

Comment:
```suggestion

    }
```

Reply:
I don't think I understand, can you elaborate on this please?

Reply:
Not sure what this was. Resolving.